source/features/quick-mention.tsx

```diff
@@ -75,7 +75,7 @@ function add(avatar: HTMLElement): void {
 		}
 	}
 
-	if (debug) {
+	if (debug && isOldView) {
```

Comment:
```suggestion
	if (debug) {
```


What's the point of enabling debugging if it silently does nothing? The variable does one thing, you set it to true when loading the old view exclusively. Since it doesn't work on the new view, you either don't use it or fix it so that it does work.